src/sagemaker/model.py

```diff
@@ -390,6 +390,7 @@ def register(
         marketplace_cert: bool = False,
         approval_status: Optional[Union[str, PipelineVariable]] = None,
         description: Optional[str] = None,
+        tags: Optional[List[Dict[str, Union[str, PipelineVariable]]]] = None,
```

Comment:
Please add unit tests for the new changes and also get approval from one of your team members